[2022-11-29 14:12:31,929] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: vaccine_dags.getDataToLocal manual__2022-11-29T14:12:30.925489+00:00 [queued]>
[2022-11-29 14:12:31,942] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: vaccine_dags.getDataToLocal manual__2022-11-29T14:12:30.925489+00:00 [queued]>
[2022-11-29 14:12:31,942] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 14:12:31,942] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-11-29 14:12:31,942] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-11-29 14:12:31,960] {taskinstance.py:1264} INFO - Executing <Task(PythonOperator): getDataToLocal> on 2022-11-29 14:12:30.925489+00:00
[2022-11-29 14:12:31,967] {standard_task_runner.py:52} INFO - Started process 921 to run task
[2022-11-29 14:12:31,972] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'vaccine_dags', 'getDataToLocal', 'manual__2022-11-29T14:12:30.925489+00:00', '--job-id', '1208', '--raw', '--subdir', 'DAGS_FOLDER/vaccine.py', '--cfg-path', '/tmp/tmprgbuibxp', '--error-file', '/tmp/tmp3jrz81_w']
[2022-11-29 14:12:31,973] {standard_task_runner.py:77} INFO - Job 1208: Subtask getDataToLocal
[2022-11-29 14:12:32,128] {logging_mixin.py:109} INFO - Running <TaskInstance: vaccine_dags.getDataToLocal manual__2022-11-29T14:12:30.925489+00:00 [running]> on host acb48b0a6e0d
[2022-11-29 14:12:32,187] {taskinstance.py:1431} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=vaccine_dags
AIRFLOW_CTX_TASK_ID=getDataToLocal
AIRFLOW_CTX_EXECUTION_DATE=2022-11-29T14:12:30.925489+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-29T14:12:30.925489+00:00
[2022-11-29 14:12:32,487] {python.py:175} INFO - Done. Returned value was: None
[2022-11-29 14:12:32,498] {taskinstance.py:1282} INFO - Marking task as SUCCESS. dag_id=vaccine_dags, task_id=getDataToLocal, execution_date=20221129T141230, start_date=20221129T141231, end_date=20221129T141232
[2022-11-29 14:12:32,545] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-11-29 14:12:32,723] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
